proto/commonfate/access/v1alpha1/access.proto

```diff
@@ -119,13 +119,19 @@ message Entitlement {
 
   // target_path is the hierarchical path to the particular entitlement target.
   // It is used in the Common Fate web console to display entitlements in a hierarchy view.
-  // For example, an AWS account entitlement will have a path of 
+  // For example, an AWS account entitlement will have a path of
   // `Root OU -> Parent OU -> Child OU.`
   //
   // target_path is in sorted order, from the root path node to the immediate parent of the target.
-  // 
+  //
   // The target_path does not include the target itself.
   repeated NamedEID target_path = 4;
+
+  // The policy ids which contributed to this entitlement being available to request
+  repeated string request_action_policy_ids = 5;
```

Comment:
The intention with these is to keep the API response light weight

There will be a seperate detail API which will be used to fetch the exact policy diagnostics